.github/pull_request_template.md

```diff
@@ -26,3 +26,11 @@ _Optional: attachments like images, videos, etc._
 - [ ] Description is filled and free of optional paragraphs.
 - [ ] Adds/updates automated tests.
 
+---
+
+### UI accessibility checklist
+
+_If your PR includes UI changes, please utilize this checklist:_
+- [ ] Make sure you use the API for UI elements that support large fonts.
+- [ ] All colors are taken from Semantic Colors.
```

Comment:
We should discuss if we want to stick to the current SemanticColors type.
In new designs the designer specifies the color theme:
![Wire 2024-07-18 at 13-37-55](https://github.com/user-attachments/assets/2eb98d60-fb22-4c86-bbe3-3fa1f68a0116)
For this example you would specify: 
```
let color = WireDesign.ColorTheme.Backgrounds.onSurface
```

Reply:
I like this way, rather than colors named by us.

Reply:
I agree that we should move to a new color system since that's what we have from the designers. Does it sound good?
`All colors are taken from WireDesign.ColorTheme`